interview-questions.md

```diff
@@ -17,30 +17,40 @@ Researched answer:
 1. What is git? What is the difference between git and Github?
 
 Your answer:
-
+git is local to your machine. github is on the interwebs of internet, storing and organizing the same code created and pushed from someones local machine
 Researched answer:
+github operates on the interwebs. on remote servers. git manages your code and folders locallly.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: <, >, ==, ===, <=, >=, <==, >==, !== conditional, relational
 
 Researched answer:
-
+logical/comparison operators.things that would return a hi or low in a digital circuit. 
+ar
 1. What is the difference between an array's index and its value?
 
-Your answer:
+Your answer: 
+index is the position in the array; which can be the  the value at an index is the data: string, num, object, boolean, etc.
 
 Researched answer:
+the location of data vs its value.
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
 Your answer:
+same: can contain numbers and letters.
+string are arrays with where all of the information is stored as only one element
+different: array: can contain multiple strings, 
 
 Researched answer:
+Strings contain one data type.
```

Comment:
I would review strings and arrays. A string is a data type of its own. Line 43 when you said that a string is an array where all the info is stored as one element is not correct, so make sure to review this material so that you can give the correct answer to an interviewer in the future!